CMakeLists.txt

```diff
@@ -66,3 +58,60 @@ if (SPIRV_HEADERS_ENABLE_EXAMPLES)
   message(STATUS "Building SPIRV-Header examples")
   add_subdirectory(example)
 endif()
+
+include(GNUInstallDirs)
+add_library(${PROJECT_NAME} INTERFACE)
```

Comment:
`INTERFACE` library type available since CMake 3.0:
- https://cmake.org/cmake/help/latest/release/3.0.html#commands

`cmake_minimum_required` should be updated.

Reply:
done.